website/src/theme/Footer/index.js

```diff
@@ -35,11 +35,11 @@ function Footer() {
           <section className={styles.footerLinks}>
             {/* <a href="https://platform.icgc-argo.org/contact">Contact</a>
             <Slash /> */}
-            <a href="https://platform.icgc-argo.org/privacy">Privacy Policy</a>
+            <a href="https://www.icgc-argo.org/page/2/privacy">Privacy Policy</a>
```

Comment:
these urls look unstable, should we provide them in env instead? (doesn't have to be now, but if it's expected to change, it seems clunky to have to wait for a software release)

Reply:
Definitely a bit clunky but is it better to have several env vars just for links?
These aren't clean urls (/page/2/, /page/77/) so I'm wary of even having a base url as an env var :(